src/components/10-atoms/button/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@axa-ch/button",
+  "version": "1.0.0-beta.1",
+  "description": "> TODO: description",
+  "author": "Pattern Warriors",
+  "homepage": "https://github.com/axa-ch/patterns-library/tree/master/src/components/10-atoms/button#readme",
+  "license": "ISC",
```

Comment:
Carefull here, pelase add See Readme and then in Readme add "AXA Versicherungen AG"